Classes/BITFeedbackManager.h

```diff
@@ -76,7 +76,11 @@ typedef NS_ENUM(NSInteger, BITFeedbackObservationMode) {
   /**
    *  Triggers when the user taps with three fingers on the screen. Captures a screenshot and attaches it to the composer.
    */
-  BITFeedbackObservationModeThreeFingerTap = 2
+  BITFeedbackObservationModeThreeFingerTap = 2,
+  /**
+   * Allows both BITFeedbackObservationModeOnScreenshot and BITFeedbackObservationModeThreeFingerTap at the same time.
+   */
+   BITFeedbackObservationModeAll = 3
```

Comment:
I think this is not a problem in this specific case but generally I would advise for this to be some other, very high number, `99` or so for example. But since it's unlikely we'll add any more options here, this should be fine 🙂